EIP-005-extism-versioning-system.md

```diff
@@ -0,0 +1,39 @@
+#  Extism Versioning System
+
+## Purpose
+
+Extism has many separate release channels. There are three categories of software in the ecosystem:
+
+a. The Host SDKs
+b. The Plugin PDKs
+c. The runtime (the shared library)
```

Comment:
I think these need a line between them if you want them to be on their own lines.